spec/support/schemas/education_enrollment.json

```diff
@@ -0,0 +1,39 @@
+{
+  "$schema" : "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "required": ["data"],
```

Comment:
As EVSS could be down but vets-api up, we could add a meta object that has server status like https://github.com/department-of-veterans-affairs/vets-api/blob/0874ca62f189a7751b923809df75f52c2d2935f9/spec/support/schemas/letters.json#L64